pokedex/src/Components/Pokesprites/Pokesprites.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./Pokesprites.css";
+
+const Pokesprites = ({ pokeData, loading, infoPokemon, activePokemon }) => {
+  return (
+    <>
+      {loading ? (
+        <h1>Loading...</h1>
+      ) : (
+        <div key={activePokemon?.id}>
+          <div className="card" onClick={() => infoPokemon(activePokemon)}>
+            <h2>{activePokemon?.id}</h2>
+            <img src={activePokemon?.sprites.front_default} alt="" />
```

Comment:
* Every tag `img` should have a **descriptive** attribute `alt` (the **name** of the image is usually there) to show the `alternative` text if the image is not uploaded and it’s needed for **screen readers**. **Please, correct it everywhere in the code**